src/core/file/workers/fileProcessWorker.ts

```
@@ -33,7 +34,20 @@
   processedContent = processedContent.trim();
 
-  if (config.output.showLineNumbers) {
+  if (config.output.compress) {
+    const parseFile = await getFn_parseFile();
+    try {
+      const parsedContent = await parseFile(processedContent, rawFile.path, config);
+      if (parsedContent === undefined) {
+        logger.trace(`Failed to parse ${rawFile.path} in compressed mode. Using original content.`);
+      }
+      processedContent = parsedContent ?? processedContent;
+    } catch (error: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)